server/views/404Page.js

```diff
@@ -0,0 +1,13 @@
+export function render404Page() {
```

Comment:
TODO: Make this page better. Maybe play a random sketch every time it's rendered?